Library/Homebrew/livecheck/version.rb

```diff
@@ -0,0 +1,69 @@
+# typed: strict
+# frozen_string_literal: true
+
+module Homebrew
+  module Livecheck
+    # A formula or cask version, split into its component sub-versions.
+    #
+    # @api private
+    class LivecheckVersion
```

Comment:
```suggestion
    class Version
```

or rename the file `livecheck_version.rb` maybe?